rfcs/text/pe/pe.yml

```
@@ -0,0 +1,226 @@
+---
+- name: pe
+  title: PE Header
+  group: 2
+  description: These fields contain Windows Portable Executable (PE) metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - dll
+      - process
+  fields:
```

Since this file is for supplemental changes, we can omit most of this. Let's just make sure the structural details are in place, and get rid of the rest, like this:

```suggestion
- name: pe
  type: group
  fields:
```